commands/changed/command.js

```diff
@@ -15,7 +15,13 @@ exports.describe = "List local packages that have changed since the last tagged
 exports.builder = yargs => {
   listable.options(yargs);
 
-  return versionOptions(yargs, "changed");
+  const newYargs = versionOptions(yargs, "changed");
+  newYargs.option("include-merged-tags", {
```

Comment:
Since this was already added in `commands/version/command.js`, you don't need to duplicate it here? `versionOptions()` was designed to avoid this duplication. It is hidden, true, but it still works.